task_templates/estimators/r_binary_classification/model-metadata.yaml

```diff
@@ -0,0 +1,25 @@
+name: r_binary_classification
+type: training
+targetType: binary
+# modelID: optional
+environmentID: 610305345da4263681a176b5
```

Comment:
On staging the R environmentID is `5ea850ca1d41c8173c2feef6`

Reply:
https://staging.datarobot.com/model-registry/custom-environments/5ea850ca1d41c8173c2feef6